provider/gce/environ_network.go

```diff
@@ -231,14 +265,21 @@ func (e *environ) ReleaseContainerAddresses([]network.ProviderInterfaceInfo) err
 	return errors.NotSupportedf("container addresses")
 }
 
+func copyStrings(items []string) []string {
```

Comment:
We have copyStringSlice in cloudinit package.
It annoys me this stuff has to be copy and pasted all the time.
Just a random whine.